docker-compose.yml

```diff
@@ -1,10 +1,19 @@
 version: '3.3'
```

Comment:
@JacOng17 I changed the version to one that is (hopefully) supported for you. I tested it my end and it build OK.

Reply:
Nice, yup. I changed it on my local & building now :+1: